src/game/phase/dungeon.cpp

```diff
@@ -72,7 +86,7 @@ proto::Phase DungeonPhase::s_update() {
     // Send the position and orientation of the specified game objects.
     // Currently sending all Player objects and Constructs.
     for (auto const & o : GameObject::game_objects) {
-        if (dynamic_cast<Player*>(o.second) || dynamic_cast<Construct*>(o.second))
+        if (dynamic_cast<Player*>(o.second) || dynamic_cast<Construct*>(o.second) || dynamic_cast<Essence*>(o.second))
```

Comment:
Yeah, I agree. but I don't really know a clean way to do it, and would prefer if someone else does it in another PR. I don't think having a variable in GameObject is the best way to do it.